<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DigitalBeamDialogClass</class>
 <widget class="QDialog" name="DigitalBeamDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>367</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>367</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>367</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Digital Beam</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="13" column="0">
    <widget class="QLabel" name="labelSubbands">
     <property name="maximumSize">
      <size>
       <width>77</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>Subbands:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="labelAngle2">
     <property name="maximumSize">
      <size>
       <width>77</width>
       <height>25</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Angle 2:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelCoordinates">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>77</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>Coordinates:</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="labelStartTime">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>77</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>Start time:</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1" colspan="5">
    <widget class="QLineEdit" name="lineEditSubbands">
     <property name="toolTip">
      <string>Comma separated list of subbands between brackets e.g.:
[1,3,5..9,11]</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1" colspan="4">
    <widget class="QLineEdit" name="lineEditAngle2"/>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QComboBox" name="comboBoxCoordinateSystem">
     <property name="toolTip">
      <string>Coordinate system</string>
     </property>
    </widget>
   </item>
   <item row="16" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEditDuration">
     <property name="toolTip">
      <string>the duration of this beam (hhhh:mm:ss)</string>
     </property>
     <property name="inputMask">
      <string>9999:99:99; </string>
     </property>
     <property name="text">
      <string>0000:00:00</string>
     </property>
    </widget>
   </item>
   <item row="16" column="0">
    <widget class="QLabel" name="labelStartTime_2">
     <property name="maximumSize">
      <size>
       <width>77</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>Duration:</string>
     </property>
    </widget>
   </item>
   <item row="19" column="4" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBoxCancelOK">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelAngleUnits">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>Angle units:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QComboBox" name="comboBoxAngleUnits">
     <property name="toolTip">
      <string>Display units for the angles</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="4">
    <widget class="QLineEdit" name="lineEditTarget">
     <property name="minimumSize">
      <size>
       <width>110</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Target denomination</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelTarget">
     <property name="maximumSize">
      <size>
       <width>77</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>Target:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QPushButton" name="pushButtonSelectSource">
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>90</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Select source from online Simbad database</string>
     </property>
     <property name="text">
      <string>Select source</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="4">
    <widget class="QLineEdit" name="lineEditAngle1">
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelAngle1">
     <property name="maximumSize">
      <size>
       <width>77</width>
       <height>25</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Angle 1:</string>
     </property>
    </widget>
   </item>
   <item row="17" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>nr. TAB rings:</string>
     </property>
    </widget>
   </item>
   <item row="17" column="1" colspan="3">
    <widget class="QSpinBox" name="spinBoxNrTabRings">
     <property name="maximum">
      <number>12</number>
     </property>
    </widget>
   </item>
   <item row="15" column="1" colspan="3">
    <widget class="QTimeEdit" name="timeEditStartTime">
     <property name="toolTip">
      <string>Start time (hh:mm::ss) of this beam relative to the start of this observation</string>
     </property>
     <property name="displayFormat">
      <string>hh:mm:ss</string>
     </property>
    </widget>
   </item>
   <item row="18" column="1" colspan="4">
    <widget class="QLineEdit" name="lineEditTabRingSize"/>
   </item>
   <item row="18" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>TAB ring size:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>lineEditAngle1</sender>
   <signal>textEdited(QString)</signal>
   <receiver>DigitalBeamDialogClass</receiver>
   <slot>getAngle1(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditAngle2</sender>
   <signal>textEdited(QString)</signal>
   <receiver>DigitalBeamDialogClass</receiver>
   <slot>getAngle2(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>getAngle1(QString)</slot>
  <slot>getAngle2(QString)</slot>
 </slots>
</ui>
